### PR TITLE
Docs: explain how to run/debug e2e tests

### LINKED ISSUE #517: Improve e2e tests running and debugging docs
**Problem**
There is a need to improve the process of running and debugging the e2e tests, there are a lot of env variables and tricks (SYSTEM_NAMESPACE, make the test namespace to persist between run, expose RabbitMQ management UI, etc...)
That are not obvious for new folks wanting to contribute that could be documented to speed up the learning curve into this matter in this repo.

**[Persona:](https://github.com/knative/eventing/blob/main/docs/personas.md)**
Developers

**Exit Criteria**
A good and complete docs for the eventing-rabbitmq e2e tests

### PR DESCRIPTION
**Pull request by KauzClay**
# Changes
- 🎁📄 Add docs to explain how to run the e2e tests, and add some tips for debugging tests.
- No user related impact, solely for contributors.

/kind documentation

Fixes #517

### CODE REVIEW

File: test/e2e/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+## Profiling RabbitMQ
+
+RabbitMQ has a dashboard with lots of information available about the cluster that can be useful for debugging tests.
+
+1. Get the credentials out of a secret on the Kubernetes cluster where the tests are running:
+  `kubectl -n <test namespace> get secrets rabbitmqc-default-user -o json | jq -r '.data["default_user.conf"]' | base64 -d`
+2. Provided you are running your tests locally, you can port forward to get access to the RabbitMQ cluster:
+  `kubectl -n TEST-NAMESPACE port-forward rabbitmqc-server-0 8081:15672`
+3. Point your browser to http://localhost:8081 and log in with the credentials from step 1.
+  **Note:** If the credentials don't work, you can also try `guest` for both username and password.
```
Comment (gabo1208) [inline test/e2e/README.md:18]: The guest credentials should be when debugging the VHost Source tests, this because the way the cluster is been created

**Review by gabo1208 — CHANGES_REQUESTED** [review]

**Review by gabo1208 — APPROVED** [review]

**Comment by benmoss** [comment]
@KauzClay can you fix the lint warnings?

**Comment by benmoss** [comment]
/lgtm
/approve

thanks for sticking with us through the the slow feedback loops @KauzClay :smil

**Pull request merged**